### PR TITLE
add OSRAM Smart+ Mini Switch AC0251700NJ blue device

### PR DESCRIPTION
**Pull request by NicolasBoulanger**
adding AC0251700NJ device with blue color

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -2014,6 +2014,30 @@ const devices = [
             await configureReporting.batteryVoltage(endpoint1);
         },
     },
+    {
+        zigbeeModel: ['Lightify Switch Mini blue'],
```
Comment (Koenkk) [inline devices.js]: As this is identical to the `AC0251100NJ` I think we should merge it with that one, can you change https://github.com/Koenkk/zigbee-herdsman-converters/pull/768/files#diff-6c9a6acf22f90d1c6e524d9f3c5c1745R1994 to zigbeeModel: ['Lightify Switch Mini', 'Lightify Switch Mini blue'],

Comment (NicolasBoulanger) [reply devices.js]: Done 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/768/commits/cf2ae906cf9de4e00c2757d0ff7c21a7633f9fbb#diff-6c9a6acf22f90d1c6e524d9f3c5c1745R1994

File: devices.js [hunk 2]
```diff
@@ -2001,6 +2001,10 @@ const devices = [
             fz.AC0251100NJ_cmdMoveToColorTemp, fz.AC0251100NJ_cmdMoveHue, fz.AC0251100NJ_cmdMoveToSaturation,
             fz.AC0251100NJ_cmdOff, fz.AC0251100NJ_cmdMove, fz.battery_3V,
             fz.AC0251100NJ_cmdMoveToLevelWithOnOff,
+            fz.AC0251700NJ_cmdOn, fz.AC0251700NJ_cmdMoveWithOnOff, fz.AC0251700NJ_cmdStop,
```
Comment (Koenkk) [inline devices.js]: I dont think extra converters are needed anymore, can you remove them?

Comment (NicolasBoulanger) [reply devices.js]: in devices.js and fromzigbee.js so ?

Comment (NicolasBoulanger) [reply devices.js]: https://github.com/Koenkk/zigbee-herdsman-converters/pull/768/commits/7bf685f51a6525bf2c62a4644448e1e52846129f

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**